src/libs/connector.ts

```diff
@@ -0,0 +1,79 @@
+import Notify, { API } from 'bnc-notify';
+import Onboard from 'bnc-onboard';
+import getRpcUrl from 'libs/rpc';
+import { Subscriptions, API as OnboardAPI } from 'libs/types';
+
+const networkId = 4;
+const rpcUrl = getRpcUrl(networkId);
+const dappId = process.env.REACT_APP_ONBOARD_API_KEY;
```

Comment:
These are all constants right? If so, I'd write them in uppercase and snake case:
```suggestion
const NETWORK_ID = 4;
const RPC_URL = getRpcUrl(networkId);
const DAPP_ID = process.env.REACT_APP_ONBOARD_API_KEY;
```

Reply:
Fixed in 4e7b000ac48a48779622f01b7a40c52c35a50f45